scripts/download-and-run-rc.js

```diff
@@ -0,0 +1,197 @@
+    const subprocess = spawnSync('mvn',
+        [
+            '-q',
+            'dependency:get',
+            `-DrepoUrl=${TEST_REPO}`,
+            `-Dartifact=com.hazelcast:hazelcast:${HAZELCAST_TEST_VERSION}:jar:tests`,
+            `-Ddest=hazelcast-${HAZELCAST_TEST_VERSION}-tests.jar`
+        ], {
+            stdio: 'inherit',
+            shell: ON_WINDOWS
+        });
+    if (subprocess.status !== 0) {
+        console.log(`Failed download hazelcast test jar com.hazelcast:hazelcast:${HAZELCAST_TEST_VERSION}:jar:tests`);
+        process.exit(1);
+    }
+}
+
+let CLASSPATH = `hazelcast-remote-controller-${HAZELCAST_RC_VERSION}.jar${ENV_VARIABLE_SEPARATOR}\
+hazelcast-${HAZELCAST_TEST_VERSION}-tests.jar${ENV_VARIABLE_SEPARATOR}\
+test/javaclasses`;
+
+if (HAZELCAST_ENTERPRISE_KEY) {
+    if (fs.existsSync(`hazelcast-enterprise-${HAZELCAST_ENTERPRISE_VERSION}.jar`)) {
+        console.log('hazelcast-enterprise.jar already exists, not downloading from maven.');
+    } else {
+        console.log(`Downloading: hazelcast enterprise jar \
+                     com.hazelcast:hazelcast-enterprise:${HAZELCAST_ENTERPRISE_VERSION}`);
+        const subprocess = spawnSync('mvn', [
+            '-q',
+            'dependency:get',
+            `-DrepoUrl=${ENTERPRISE_REPO}`,
+            `-Dartifact=com.hazelcast:hazelcast-enterprise:${HAZELCAST_ENTERPRISE_VERSION}`,
+            `-Ddest=hazelcast-enterprise-${HAZELCAST_ENTERPRISE_VERSION}.jar`
+        ], {
+            stdio: 'inherit',
+            shell: ON_WINDOWS
+        });
+        if (subprocess.status !== 0) {
+            console.log(`Failed download hazelcast enterprise jar \
+            com.hazelcast:hazelcast-enterprise:${HAZELCAST_ENTERPRISE_VERSION}`);
+            process.exit(1);
+        }
+    }
+
+    if (fs.existsSync(`hazelcast-enterprise-${HAZELCAST_TEST_VERSION}-tests.jar`)) {
+        console.log('hazelcast-enterprise-tests.jar already exists, not downloading from maven.');
+    } else {
+        console.log(`Downloading: hazelcast enterprise test jar \
+                     com.hazelcast:hazelcast-enterprise:${HAZELCAST_TEST_VERSION}:jar:tests`);
+        const subprocess = spawnSync('mvn', [
+            '-q',
+            'org.apache.maven.plugins:maven-dependency-plugin:2.8:get',
+            `-DrepoUrl=${ENTERPRISE_TEST_REPO}`,
+            `-Dartifact=com.hazelcast:hazelcast-enterprise:${HAZELCAST_TEST_VERSION}:jar:tests`,
+            `-Ddest=hazelcast-enterprise-${HAZELCAST_TEST_VERSION}-tests.jar`
+        ], {
+            stdio: 'inherit',
+            shell: ON_WINDOWS
+        });
+        if (subprocess.status !== 0) {
+            console.log(`Failed to download hazelcast enterprise test jar \
+                         com.hazelcast:hazelcast-enterprise:${HAZELCAST_TEST_VERSION}:jar:tests`);
+            process.exit(1);
+        }
+    }
+    CLASSPATH = `hazelcast-enterprise-${HAZELCAST_ENTERPRISE_VERSION}.jar${ENV_VARIABLE_SEPARATOR}\
```

Comment:
Long lines with template literals like this one could be split in multiple literals in order to make them more readable. Example:
```js
    CLASSPATH = `hazelcast-enterprise-${HAZELCAST_ENTERPRISE_VERSION}.jar${ENV_VARIABLE_SEPARATOR}\`
        + `hazelcast-enterprise-${HAZELCAST_ENTERPRISE_TEST_VERSION}-tests.jar${ENV_VARIABLE_SEPARATOR}\`
        + CLASSPATH;
```